com.unity.probuilder/Documentation~/auto-uvs-actions.md

```diff
@@ -57,19 +57,18 @@ By default this is set to **None**, but you can select any of the following poin
 |![G](images/LetterCircle_G.png) __Lower Left__ |![H](images/LetterCircle_H.png) __Lower Center__ |![I](images/LetterCircle_I.png) __Lower Right__ |
 
 
-Anchor points represent a point on the original UV face (before it was scaled or tiled).
-
+Anchor points represent a point on the original UV face (before you scale or tile it).
```

Comment:
This is no longer accurate as of `4.0.4`. Anchor is now applied after Rotation and Scaling (Tiling).